androidApp/.gitignore

```diff
@@ -0,0 +1,2 @@
+/build
+google-services.json
```

Comment:
This is already in the main directory's .gitignore. Do we want to remove it from there?

Reply:
Ah, missed that. I'll close this then